packages/astro/src/core/errors/errors-data.ts

```diff
@@ -530,8 +530,22 @@ export const MissingImageDimension = {
 	message: (missingDimension: 'width' | 'height' | 'both', imageURL: string) =>
 		`Missing ${
 			missingDimension === 'both' ? 'width and height attributes' : `${missingDimension} attribute`
-		} for ${imageURL}. When using remote images, both dimensions are always required in order to avoid CLS.`,
-	hint: 'If your image is inside your `src` folder, you probably meant to import it instead. See [the Imports guide for more information](https://docs.astro.build/en/guides/imports/#other-assets).',
+		} for ${imageURL}. When using remote images, both dimensions are required unless in order to avoid CLS.`,
+	hint: 'If your image is inside your `src` folder, you probably meant to import it instead. See [the Imports guide for more information](https://docs.astro.build/en/guides/imports/#other-assets). You can also use `inferSize={true}` for remote images to get the original dimensions.',
+} satisfies ErrorData;
+/**
+ * @docs
+ * @message
+ * Failed to get the dimensions for `IMAGE_URL`.
+ * @description
+ * Probing the remote image's dimensions failed, this is typically caused by an incorrect URL or attempting to infer the size of an image in the public folder. Infering the size of images in public folder is not possible.  It is also possible that you are trying to probe a file type that is not supported. Please create [an issue](https://github.com/withastro/astro/issues) if you want to see a filetype that is not currently supported be added. All file types supported by Sharp are supported, as well as .bmp, .ico, .heic and .heif. Here is where the probing logic lives in Astro's [sourcecode](https://github.com/withastro/astro/blob/main/packages/astro/src/assets/utils/remoteProbe.ts).
+ */
+export const FailedToProbeRemoteImage = {
+	name: 'FailedToProbeRemoteImage',
+	title: 'Failed to probe remote image dimensions',
+	message: (imageURL: string) =>
+		`Failed to get the dimensions for ${imageURL}.`,
+	hint: 'Verify your image URL is accurate. If you are using an image in public folder, you will not be able to infer its size. It is also possible that you are trying to probe a file type that is not supported. Please create [an issue](https://github.com/withastro/astro/issues) if you want to see a filetype that is not currently supported be added.',
```

Comment:
```suggestion
	hint: 'Verify your remote image URL is accurate, and that you are not using `inferSize` with a file located in your `public/` folder. It is also possible that you are trying to get the dimensions of a file type that is not supported.',
```
Again, I don't really think of "you can file an issue" as a hint for how to solve your problem, so I'd suggest removing UNLESS you are really waiting for people to ask for more stuff so you can build it.  :smile: You could consider listing the file types again here, like the message before.

(Again,